configs/gvisor-containerd/config.toml

```diff
@@ -5,6 +5,8 @@ state = "/run/gvisor-containerd"
   address = "/run/gvisor-containerd/containerd.sock"
 [plugins."io.containerd.runtime.v1.linux"]
   shim_debug = true
+[plugins."io.containerd.grpc.v1.cri".containerd.runtimes.runc]
+  runtime_type = "io.containerd.runc.v1"
```

Comment:
as non-function components should run in the regular runc containers, I am not sure what these 2 lines are. I'd try removing them. 

Also, please check that only functions and nothing else runs in gvisor sandbox (top on the nodes would do to spot gvisor processes).

Reply:
> as non-function components should run in the regular runc containers, I am not sure what these 2 lines are. I'd try removing them.

Removing these lines does not help (actually, I added these lines to try to make it work).

> Also, please check that only functions and nothing else runs in gvisor sandbox (top on the nodes would do to spot gvisor processes).

Not sure what you mean, but I filtered out the processes by `gvisor` and there's only `gvisor-containerd` running:
<img width="614" alt="Screenshot 2023-03-08 at 15 27 24" src="https://user-images.githubusercontent.com/32464072/223713262-68719ae9-ee9a-43d5-8389-cc59e710717b.png">
<img width="745" alt="Screenshot 2023-03-08 at 15 21 05" src="https://user-images.githubusercontent.com/32464072/223713305-ae96d3b4-5d18-4ba4-a929-dd7ff36fcc71.png">